components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -2005,6 +2005,23 @@ document.observe('xwiki:dom:loaded', function() {
       new PhenoTips.widgets.ChartGenerator($('charts'), chartDataSources);
     });
 
+    //create a warning when measurements are being entered buyt no birth date is specified
+    var dateOfBirth=$(document.documentElement).down('input[name$=date_of_birth]');
+    var previousElement= $$('.measurement-info &gt; .expand-tools')[0] ? $$('.measurement-info &gt; .expand-tools')[0] : $('HMeasurements');
```

Comment:
`previousElement` doesn't say much. Previous to what? Better name: `warningInsertionPoint`.